src/gt4py/next/ffront/fbuiltins.py

```diff
@@ -95,6 +95,8 @@ def __call__(self, *args: _P.args, **kwargs: _P.kwargs) -> _R:
 
     def dispatch(self, *args: Any) -> Callable[_P, _R]:
         arg_types = tuple(type(arg) for arg in args)
+        if any(t == tuple for t in arg_types):
+            return self.function
```

Comment:
obviously this is ugly, @tehrengruber do you want to discuss?

Reply:
Till's temporary idea: 
try gt_builtin_func from each argument, if all fail, we do the default implementation